client/src/components/DataFiles/DataFilesModals/DataFilesDownloadMessageModal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import './DataFilesDownloadMessageModal.module.scss';
+
+const DataFilesDownloadMessageModal = () => {
+  const isOpen = useSelector(state => state.files.modals.downloadMessage);
+
+  const dispatch = useDispatch();
+  const toggle = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'downloadMessage', props: {} }
+    });
+  };
+
+  return (
+    <Modal isOpen={isOpen} toggle={toggle} size="md" className="dataFilesModal">
+      <ModalHeader toggle={toggle}>Download</ModalHeader>
+      <ModalBody>
+        <p>Folders must be compressed before download.</p>
+        <ol>
+          <li>Select the folder(s).</li>
+          <li>Press the &quot;↩ Compress&quot; icon above the table.</li>
+          <li>Complete and submit the form.</li>
+          <li>After the job finishes, you may download the compressed file.</li>
+        </ol>
+      </ModalBody>
+      <ModalFooter>
+        <Button type="button" color="primary" onClick={toggle}>
+          Close
+        </Button>
```

Comment:
I think this `<Button>` needs the `.c-button--primary` class -- could you import the class and compose it into a style for this button in `DataFilesDownloadMessageModal.module.scss`?
```
.button {
  composes: c-button--primary from '../../../styles/components/c-button.css';
}
```